waik/waik_finish.erb

```diff
@@ -0,0 +1,98 @@
+<%#
+#kind: finish
+#name: WAIK default
+#oses:
+#- Windows 6.1
+#- Windows 6.3
+# Parameters are expected to be set in Foreman (globally or per group/host)
+params:
+- windowsLicenseKey: ABCDE-ABCDE-ABCDE-ABCDE-ABCDE # Valid Windows license key
+- windowsLicenseOwner: Company, INC # Legal owner of the Windows license key
+- localAdminiAccountDisabled: false
+- ntpSever: time.windows.com,other.time.server
+- domainAdminAccount: administrator@domain.com # use this account to join domain
+- domainAdminAccountPasswd: Password for the domain Admin account
```

Comment:
@sathieu This dependents on how your domain is configured and witch [GPOs are in place](https://support.microsoft.com/en-us/kb/243327). A domain admin always work (for instance, my users may not add any computer to the domain...)

About these or any other parameters: To keep I will refactor them one we're a point near closing merging this PR ;)